### PR TITLE
contributing my feature wrappers for https://github.com/nvbn/thefuck 

### PR DESCRIPTION
**Pull request by GeorgOfenbeck**
Heavily inspired by @danielbraun89 implementations

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -668,3 +668,8 @@
   contact: https://github.com/rsm-hcd
   repository: https://github.com/rsm-hcd/devcontainer-templates
   ociReference: ghcr.io/rsm-hcd/devcontainer-templates
+- name: Additional Dev Container Features by Georg Ofenbeck 
+  maintainer: Georg Ofenbeck
+  contact: https://github.com/GeorgOfenbeck/features/issues
+  repository: https://github.com/GeorgOfenbeck/features 
+  ociReference: ghcr.io/GeorgOfenbeck/features
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/georgofenbeck/features
```

Hi 👋

The ORAS spec requires the `ociReference` to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Left one minor comment!

File: _data/collection-index.yml (hunk 1 above)
Comment (GeorgOfenbeck) [reply _data/collection-index.yml]: of course! thanks

**Review by samruddhikhandale — DISMISSED** [review]
Looks good, thanks!

**Review by samruddhikhandale — APPROVED** [review]

**Pull request merged**